pytorch_lightning/strategies/sharded.py

```diff
@@ -140,3 +259,49 @@ def register_strategies(cls, strategy_registry: Dict) -> None:
             cls,
             description=f"{cls.__class__.__name__}",
         )
+
+    def reconciliate_processes(self, trace: str) -> None:
+        if self.world_size < 2:
+            return
+
+        if not self._should_run_deadlock_detection():
```

Comment:
where is this defined? this is defined in DDP here: https://github.com/PyTorchLightning/pytorch-lightning/blob/cf64f3443474a93d23b5afb0417e4a60298006e6/pytorch_lightning/strategies/ddp.py#L352-L359